### DISCUSSION TITLE
[Esame 22 gennaio 2024] Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#445] [Esami M-Z]
![22 gennaio es.4](https://github.com/user-attachments/assets/ed63fd3f-3444-466b-9111-7e2fed6e9a3b)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/61c3f8e7-e8a9-44e1-8a06-2ac1c677f32a" alt="soluzione"  />
  
</details>

**Comment by AglaiaNorza** [comment]
Soluzione ad opera della [tutor](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md#aa-202425-scandiuzzi-tutor) Elisa Scandiuzzi, lezione del 15 ottobre 2024 (`appunti_24_10_15.pdf`):
 
![es.4](https://github.com/user-attachments/assets/265baa08-edd3-4252-8c60-36b4138dac61)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/53b71141-afe5-4d3d-9eab-36f8fc50ad08)

**Comment by niccolo-fato** [comment]
<img width="1280" height="824" alt="image" src="https://github.com/user-attachments/assets/0dc25832-b543-47e9-8116-7739ae93cb62" />
